dockerfiles/README.md

```diff
@@ -83,24 +83,79 @@
 
-1. Build the docker image from the Dockerfile in this repository.
-  ```
-  # If you have a Linux machine, preface this command with "sudo"
-
-  docker build -t onnxruntime-openvino -f Dockerfile.openvino .
-  ```
-  To use GPU_FP32:
-  ```
-  docker build -t onnxruntime-openvino --build-arg TARGET_DEVICE=GPU_FP32 -f Dockerfile.openvino .
-  ```
-
-2. Run the Docker image
-
-  ```
-  # If you have a Linux machine, preface this command with "sudo"
-
-  docker run -it onnxruntime-openvino
-  ```
+1. Build the onnxruntime image for all the accelerators supported as below 
+
+   Retrieve your docker image in one of the following ways.
+
+    -  For building the docker image, download OpenVINO online installer version 2018 R5.0.1 from [here](https://software.intel.com/en-us/openvino-toolkit/choose-download) and copy the openvino tar file in the same directory and build the image. The online installer size is only 16MB and the components needed for the accelerators are mentioned in the dockerfile.
+       ```
+       docker build -t onnxruntime -$device --build-arg DEVICE=$DEVICE .
+       ```
+    - Pull the official image from DockerHub.
+   
+
+2. DEVICE: Specifies the hardware target for building OpenVINO Execution Provider. Below are the options for different Intel target devices.
+
+	| Device Option | Target Device |
+	| --------- | -------- |
+	| <code>CPU_FP32</code> | Intel<sup></sup> CPUs |
+	| <code>GPU_FP32</code> |Intel<sup></sup> Integrated Graphics |
+	| <code>GPU_FP16</code> | Intel<sup></sup> Integrated Graphics |
+	| <code>MYRIAD_FP16</code> | Intel<sup></sup> Movidius<sup>TM</sup> USB sticks |
+	| <code>VAD-R_FP16</code> | Intel<sup></sup> Vision Accelerator Design based on Movidius<sup>TM</sup> MyriadX VPUs |
+
+## CPU Version 
+
+1. Retrieve your docker image in one of the following ways.
+
+   - Build the docker image from the DockerFile in this repository. Providing the argument device enables onnxruntime for that particular device. You can also provide arguments ONNXRUNTIME_REPO and ONNXRUNTIME_BRANCH to test that particular repo and branch. Default values are http://github.com/microsoft/onnxruntime and repo is master
+      
+     ```
+     docker build -t onnxruntime-cpu --build-arg DEVICE=CPU_FP32 --network host .
+     ```
+   - Pull the official image from DockerHub.
+     ```
+     # Will be available with next release
+     ```
+2. Run the docker image
+    ```
+     docker run -it onnxruntime-cpu
+    ```
+
+## GPU Version
+
+1. Retrieve your docker image in one of the following ways. 
+   - Build the docker image from the DockerFile in this repository.
+     ``` 
+      docker build -t onnxruntime-gpu --build-arg DEVICE=GPU_FP32 --network host . 
+     ```
+   - Pull the official image from DockerHub.
+     ```
+       # Will be available with next release
+     ```
+
+2. Run the docker image
+    ```
+    docker run -it --device /dev/dri:/dev/dri onnxruntime-gpu:latest
+    ```
+
+## VAD-R Accelerator Version 
```

Comment:
missing steps for building MYRIAD_FP16

Reply:
Will update it by today

Reply:
I have updated these and pushed the latest change